node/src/chain_specs/manta/mod.rs

```diff
@@ -50,155 +68,79 @@ pub fn manta_properties() -> Properties {
     p
 }
 
+/// Returns the Manta mainnet chainspec
```

Comment:
There are now two `manta_mainnet_config()` functions... might want to rename one